Cargo.toml

```diff
@@ -22,7 +22,7 @@ members = [
 resolver = "2"
 
 [patch.crates-io]
-xtra = { git = "https://github.com/Restioson/xtra", rev = "7e9fccc1e02dae3df4ba99126e6f1a0059673d7e" } # Unreleased
+xtra = { git = "https://github.com/Restioson/xtra", rev = "afeffb7155f02ca81f84d5d74e096fca906dae69" } # TODO Unreleased
```

Comment:
this adds dependency on a side branch again? is there a chance to avoid it? if it's absolutely necessary, I'd prefer to be explicit about it (ie. `branch = span-late-create`), but only if there's no reasonable chance to merge that soon.

Reply:
In an ideal world, we could get https://github.com/Restioson/xtra/pull/120 merged sooner rather than later and then get this PR in after, pinning xtra again a commit on master which includes the changes from that PR

Reply:
Side note: this will technically compile without https://github.com/Restioson/xtra/pull/120, but `send(..).instrument(span)` will not work right.